### ISSUE TITLE
Suggestion Arrows

### ISSUE THREAD

**Issue by Paralusion** [THEREALWWEFAN231/wwe#511]
**Minecraft Version**
All
**What is the suggestion**
Draws arrows on screen that points on players that are out of your field of view
**Why it will be useful**
Allows you to know where people are, even if you can't see them directly. The arrows dissapear when the player is out of renderdistance.

**Comment by dandud100** [comment]
So basically: PlayerESP mode: lines

**Comment by Paralusion** [comment]
correct, but more like tracers, except they indicate what direction the player is in
